ddocs/medic-client/views/total_clinics_by_facility/map.js

```diff
@@ -1,33 +1,20 @@
 function(doc) {
   if (doc.type === 'clinic') {
-    var dh = doc.parent && doc.parent.parent,
-        dh_id = dh && dh._id,
-        dh_name = dh && dh.name,
-        hc_id = doc.parent && doc.parent._id,
-        hc_name = doc.parent && doc.parent.name,
-        cl_name = doc.name,
-        phone = doc.contact && doc.contact.phone,
-        placeId = doc.place_id || (doc.contact && doc.contact.rc_code);
 
-    if (!cl_name && doc.contact) {
-      if (doc.contact.name) {
-        cl_name = doc.contact.name;
-        if (placeId) {
-          cl_name += ' ' + placeId;
-        }
-      } else {
-        cl_name = placeId;
+    var districtId = doc.parent && doc.parent.parent && doc.parent.parent._id,
```

Comment:
Yeah I've had a look for that before but couldn't find a nice way to include common libraries in Pouch views. I might've missed something but for now I'm stuck inlining everything  :/